ee-9/feature-pack/src/main/resources/modules/system/layers/base/org/wildfly/transaction/client/main/module.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+~ JBoss, Home of Professional Open Source.
+~
+~ Copyright 2021 Red Hat, Inc., and individual contributors
+~ as indicated by the @author tags.
+~
+~ Licensed under the Apache License, Version 2.0 (the "License");
+~ you may not use this file except in compliance with the License.
+~ You may obtain a copy of the License at
+~
+~     http://www.apache.org/licenses/LICENSE-2.0
+~
+~ Unless required by applicable law or agreed to in writing, software
+~ distributed under the License is distributed on an "AS IS" BASIS,
+~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+~ See the License for the specific language governing permissions and
+~ limitations under the License.
+-->
+
+<module xmlns="urn:jboss:module:1.9" name="org.wildfly.transaction.client">
+
```

Comment:
I think this module should have its own name that reflects its GAV. i.e., not the same as the existing one.

Reply:
If you change the module name then any other module that depends on this needs to have its module.xml forked. And any project that build on WF that depend on it would be broken.

Ideally for cases where the module.xml in WF Preview only differs because of a different maven artifactId or classifier we could just put an expression in the std wildfly module.xml and then at build time resolve the expression to build with the appropriate artifact. Either via maven-resource-plugin filtering or https://issues.redhat.com/browse/WFGP-206.

Reply:
okay, let's stick with the original name for now.